etcdutl/etcdutl/backup_command.go

```diff
@@ -0,0 +1,331 @@
+// Copyright 2015 The etcd Authors
```

Comment:
Should we change this as well, as while some things were copied a lot of the code changed?

Reply:
DONE